frontend/tests/integration/components/people-skill-show-test.js

```diff
@@ -21,7 +21,6 @@ module("Integration | Component | people-skill-show", function(hooks) {
 
     let text = this.$().text();
 
-    assert.ok(text.includes("Rails"));
```

Comment:
Why was this just removed?

Reply:
The Skill title is now an optional feature and not loaded in the way that this component is tested